tests/testthat/test-loop_tasks.R

```diff
@@ -20,8 +20,8 @@ test_that("can run loop_tasks to completion even when tasks fail sometimes", {
   # with verbose=TRUE, should see errors but also completion
   set.seed(100)
   options('scipiper.test_verbose'=TRUE)
-  output <- capture_messages(scmake('models.ind'))
-  expect_true(all(file.exists('models.ind','AZ.ind','CA.ind','CO.ind')), info='all files get created')
+  output <- capture_messages(scmake('tmp/models.ind'))
+  expect_true(all(file.exists('tmp/models.ind','AZ.ind','CA.ind','CO.ind')), info='all files get created')
```

Comment:
Now that `ind_dir='tmp'` in the `create_task_plan()` call above, I'm guessing that `AZ.ind`, etc. get written to `tmp/AZ.ind`, etc. - is that the case? If so then this test should change to correspond to the new `ind_dir`.

Reply:
oh, I thought I made that change. Maybe I didn't push to master...